src/Runner.js

```diff
@@ -271,15 +242,17 @@ export default class Runner {
     }
   }
 
-  getState() {
+  getState(): RunnerState {
     return {
       queue: this.queue.list(),
       running: Object.assign({}, this.running),
       waitingToLand: this.waitingToLand,
       locked: this.locked,
-      started: this.started,
+      started: String(this.started),
       paused: this.paused,
-      pausedReason: this.pausedReason
+      pausedReason: this.pausedReason,
+      history: this.history.take(10),
+      usersAllowedToMerge: this.config.prSettings.usersAllowedToApprove
```

Comment:
I added that back, need it in UI :)

Reply:
We can fetch this with the settings endpoint.
Will do that in a follow up though I think.

Reply:
Oh, right, you told me about settings endpoint, and I totally forgot about it :( sorry